src/examples/form/form.tsx

```diff
@@ -1,60 +1,116 @@
+import { ValidationStatus } from '@limetech/lime-elements';
 import { Component, h, State } from '@stencil/core';
 
 const schema = {
-    title: 'My form',
-    description: 'This is a description text for the form.',
+    title: 'Registration form',
+    description: 'Please enter your personal information',
     type: 'object',
-    required: ['title'],
+    required: ['name', 'email', 'newsletters'],
     properties: {
-        title: {
+        name: {
             type: 'string',
-            title: 'Title',
-            default: 'A new task',
-            description: 'Enter a title',
+            title: 'Name',
+            default: 'Batman',
+            description: 'Enter your heroic name',
+            minLength: 5,
+            maxLength: 20,
         },
-        done: {
-            type: 'boolean',
-            title: 'Done?',
-            default: false,
-            required: true,
+        email: {
+            type: 'string',
+            title: 'Email',
+            description: 'Enter your email address',
+            format: 'email',
+        },
+        date: {
+            type: 'string',
+            title: 'Birthdate',
+            format: 'date',
+            description: 'Pick a nice date!',
         },
         color: {
             type: 'string',
-            title: 'Color',
-            enum: ['red', 'green', 'blue'],
+            title: 'Favorite cape color',
+            description: 'Preferably matching your boots',
+            oneOf: [
+                {
+                    type: 'string',
+                    const: 'red',
```

Comment:
Is the property name `const` decided by the form-lib we're using?

Reply:
It's part of the JSON schema spec

Reply:
Yeah, I assumed so. Then I won't pester you with complaints about using a reserved word as a property name 😄

(I'm pretty surprised nothing in our lint-and-build chain balks at that…)